### DISCUSSION TITLE
PrimeReact v10.7.0 broke dropdowns

### DISCUSSION THREAD

**Discussion by jnlinn** [primefaces/community#2273] [PrimeReact] [answered]
I just tried to update from PrimeReact 10.6.6 to 10.7.0 and it "broke" all my dropdowns that contain optionValue. I assume this is due to fix #6058

Is there something I need to modify on my end to make dropdowns function?

**Comment by melloware** [comment, marked as answer]
i would stay on 10.6.6 until next release as its fixed for 10.7.1: https://github.com/primefaces/primereact/issues/6822
